wrapanapi/systems/rhevm.py

```diff
@@ -442,6 +442,8 @@ def add_disk(self, storage_domain=None, size=None, interface='VIRTIO', format=No
             interface: string disk interface type
             format: string disk format type
             active: boolean whether the disk is active
+            sparse: boolean whether the disk is preallocated or thin-provisioned
```

Comment:
do you think its better to add some info reflecting `sparse=thin provisioning` for those who may not know?

Reply:
I can state something like `... is preallocated (False) or thin-provisioned (True)...`, but I don't want to re-state the ovirt API here.

Reply:
I understand your concern, but since you are using `preallocated or thin-provision` and `sparse` all on the same line, thought it would be better just to mention something like `thin-prov=sparse`